### DISCUSSION TITLE
support for pfBlockerNG devel pfsense

### DISCUSSION THREAD

**Discussion by pinkworld123** [blocklistproject/Lists#866] [Ideas]
Lists are not working for pfBlockerNG devel, for example below links are not working. 

https://raw.githubusercontent.com/blocklistproject/Lists/master/torrent.txt
https://blocklistproject.github.io/Lists/torrent.txt
https://blocklistproject.github.io/Lists/alt-version/torrent-nl.txt
https://blocklistproject.github.io/Lists/dnsmasq-version/torrent-dnsmasq.txt
https://blocklistproject.github.io/Lists/adguard/torrent-ags.txt

I tried this all urls but none are working with pfBlockerNG devel. 

Please support this. thanks

**Comment by spirillen** [comment]
Try to look in your logs, you will probably notice that you can't use anything hosted over secure connections. I have reported this to pfsense but they closed the issue as they are not going to support HTTPs

**Comment by pinkworld123** [comment]
Thanks for reply. 

I have already tried https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/fakenews-gambling-porn/hosts

this works fine its also https. I think it should be some syntax error in blocklistproject list.

**Comment by spirillen** [comment]
which module do you try to add the lists, there are several options in pfsense

a link to the place in the docs will be helpful to all

**Comment by pinkworld123** [comment]
@spirillen module/plugin  pfBlockerNG devel

thanks

**Comment by spirillen** [comment]
> module/plugin pfBlockerNG devel

I mean whic of pfsences modules

firewall
dns
blacklist
proxy blacklist
?? which one post me a link and I spend a minute on it for you

**Comment by gap579137** [comment]
Were you able to get this resolved? I would be more than happy to create a version for PFsense
